test.sh

```diff
@@ -45,7 +45,18 @@ for sample in $(ls $INPUT); do
   rm -f measurements.txt
   ln -s $sample measurements.txt
 
-  diff <("./calculate_average_$FORK.sh") ${sample%.txt}.out
+  if command -v git; then
+    # git version must be 2.43 or newer to use \`git diff\` with process substitution
+    # Source: https://stackoverflow.com/questions/22706714/why-does-git-diff-not-work-with-process-substitution
+    # In the future, we can do this instead of a temp file:
+    #   git diff --no-index --word-diff <("./calculate_average_$FORK.sh") ${sample%.txt}
+    temp_file=$(mktemp)
+    "./calculate_average_$FORK.sh" > $temp_file
+    git diff --no-index --word-diff $temp_file ${sample%.txt}.out
+    rm $temp_file
+  else
+    diff <("./calculate_average_$FORK.sh") ${sample%.txt}.out
+  fi
```

Comment:
I've added tocsv.sh to make this easier https://github.com/gunnarmorling/1brc/pull/36
but did not use in test.sh 

It would be something like:
```
diff <("./calculate_average_$FORK.sh" | ./tocsv.sh) <(./tocsv.sh < ${sample%.txt}.out)
```

Reply:
Adding `tocsv.sh` to `git diff` is really nice 😄 

<img width="415" alt="image" src="https://github.com/gunnarmorling/1brc/assets/91577/fea00745-96b4-4624-8bd6-28ec980c6141">

Reply:
However, I'm going to simplify this to `diff <("./calculate_average_$FORK.sh" | ./tocsv.sh) <(./tocsv.sh < ${sample%.txt}.out)` to avoid the complexity of the git process substitution stuff

Reply:
...pending Gunnar's answer to my latest question